presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -45,6 +45,8 @@
     private Duration remoteTaskMinErrorDuration = new Duration(2, TimeUnit.MINUTES);
     private int remoteTaskMaxCallbackThreads = 1000;
 
+    private Duration queryMaxRuntime = new Duration(1, TimeUnit.HOURS);
```

Comment:
We also have several queries that run more than several hours. Please make the default value 1 day.